docs/developers/instantiate-testnet.md

```diff
@@ -0,0 +1,244 @@
+# Celestia App Network Instantiation Guide
+
+This guide is for helping instantiate a new testnetwork and following the
+correct steps to do so with Celestia-App. You should only follow this guide
+if you want to experiment with your own Celestia Testnetwork or if you want
+to test out new features to build as a core developer.
+
+## Hardware Requirements
+
+The following hardware minimum requirements are recommended for running the
+validator node:
+
+* Memory: 8 GB RAM
+* CPU: Quad-Core
+* Disk: 250 GB SSD Storage
```

Comment:
Someone else can confirm, but IIRC Cosmos apps are really bloated and even today that 250 GB will go fast.

Reply:
this obviously depends on pruning, tx indexing, and chain usage, but 250GB should a safe bet for a few months using the default settings. This might end up being a very safe estimate considering that our block times are 3-5 longer

when pruning everything, and not indexing txs, 120GB is a safe bet for a few months, again this depends significantly on chain usage.